napari_allencell_annotator/_tests/utils/test_file_utils.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+from typing import List
+
+from napari_allencell_annotator.util.file_utils import FileUtils
+import napari_allencell_annotator
+
+
+def test_sorted_dirs_and_files_in_dir():
+    # ACT
+    sorted_file_list: List[Path] = FileUtils.get_sorted_dirs_and_files_in_dir(
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "sort_img_dir"
+    )
+
+    # ASSERT
+    assert sorted_file_list == [
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "sort_img_dir" / "a",
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "sort_img_dir" / "b.ome.tiff",
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "sort_img_dir" / "c.ome.tiff",
+    ]
+
+
+def test_select_valid_images() -> None:
+    # ARRANGE
+    all_files: List[Path] = [
+        Path("test.tiff"),
+        Path("test.png"),
+        Path("test.jpeg"),
+        Path("test.czi"),
+        Path("test/raw.ome.zarr"),
+        Path("test.csv"),
+        Path("test.json"),
+        Path("valid_img_dir"),
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "zarr_dir",
+    ]
+
+    # ACT
+    valid_files: List[Path] = FileUtils.select_valid_images(all_files)
+
+    # ASSERT
+    assert valid_files == [
+        Path("test.tiff"),
+        Path("test.png"),
+        Path("test.jpeg"),
+        Path("test.czi"),
+        Path("test/raw.ome.zarr"),
+        Path(napari_allencell_annotator.__file__).parent / "_tests" / "assets" / "zarr_dir" / "raw.ome.zarr",
+    ]
+
+
+def test_get_file_name() -> None:
+    # ACT
+    tiff_file_name: str = FileUtils.get_file_name(Path("parent/test.tiff"))
+    png_file_name: str = FileUtils.get_file_name(Path("parent/test.png"))
+    jpeg_file_name: str = FileUtils.get_file_name(Path("parent/test.jpeg"))
+    czi_file_name: str = FileUtils.get_file_name(Path("parent/test.czi"))
+    zarr_file_name: str = FileUtils.get_file_name(Path("parent/test.ome.zarr"))
+
+    # ASSERT
+    assert tiff_file_name == "test.tiff"
+    assert png_file_name == "test.png"
+    assert jpeg_file_name == "test.jpeg"
+    assert czi_file_name == "test.czi"
+    assert zarr_file_name == "parent"
```

Comment:
This is fine, but the only two cases you really need to cover (for this test) is one image and one zarr, all the file types should behave in the same way. We only need to test all of our test files for file opening

Reply:
dont need to change it since you already have it

Reply:
got it